lib/vc.js

```diff
@@ -213,10 +213,11 @@ async function _verifyCredential(options = {}) {
 /**
  * Creates an unsigned presentation from a given verifiable credential.
  *
- * @param {object|Array<object>} verifiableCredential - One or more
- *   verifiable credential
- * @param {string} [id] - Optional VP id
- * @param {string} [holder] - Optional presentation holder url.
+ * @param {object} options - Options to use.
```

Comment:
Why this change? There's no actual parameter named `options`...

Reply:
When a function takes an object as the parameters jsdoc requires it be documented. So basically from jsdoc's PoV this is a function with one argument: options and then options should have the properties specified in it. It is kind of weird and it would be nice if we could extend js-doc and it could infer we're using an object so we have named arguments to a function.

Reply:
@aljones15 ahh got it, thanks! (For some reason I thought jsdoc was smart enough to handle it).

Reply:
@dmitrizagidulin as a general rule jsdoc is not that smart and if you've ever tried to actually export it to something like say markdown or an api specification you are in for some pain.